plugins/catalog/src/components/EntityContextMenu/EntityContextMenu.tsx

```diff
@@ -84,6 +86,24 @@ export function EntityContextMenu(props: EntityContextMenuProps) {
     setAnchorEl(undefined);
   };
 
+  const [state, setState] = useState({
+    open: false,
+    vertical: 'top',
+    horizontal: 'right',
```

Comment:
Why is the placement in the state? I mean it can be, but seems like it should just be constant; and you don't even read these in the rendering below when setting the `anchorOrigin`

Reply:
Removed, it's constant